canvas/cli/common.py

```diff
@@ -1,35 +1,51 @@
+import json
 import sys
 
+DEFAULT_JSON = False
+DEFAULT_TABLE = False
+
 # keys: [(items key, title, pretty title), ...]
-def cli_list(items, keys, table = False, skip_headers = False,
-        collective_name = 'items', sort_key = 'id'):
+def cli_list(items, keys, table = DEFAULT_TABLE, skip_headers = False,
+        collective_name = 'items', sort_key = 'id', output_json = DEFAULT_JSON):
     if (len(items) == 0):
         print("No %s found." % (collective_name), file = sys.stderr)
         return 0
```

Comment:
I'm not a fan of this return for JSON.
For the text-based formats it's not an issue,
but for JSON is is not formatted correctly (i.e., it will break programs who are using this one's output.

(I know it is outputting to stderr, but then we have an empty stdout which is still not a JSON list.)